script/Arcueid/Day 13/13_00_ARC13_2.txt

```diff
@@ -241,7 +257,10 @@ What drives her now is the overwhelming joy that surrounds her.
 [sha:29116569eaa8e39e432cac0f4b7709b4930a14ce]{
 -- Page 229, Offset 32298.
 -- ああ、と瞳が、唇が、全身がはにかんでいる。
-Her eyes, her lips, and her entire body grows abashed as she lets out her voice.
+// Reading this ああ as a squee, with associated toe-curling excitement / embarrassment?
+// Possibly overwritten but felt a bit awkward literal
```

Comment:
I think this is a tad overwritten but at the same time I think this is a perfectly fair localization.